src/openrct2-ui/windows/ScenarioSelect.cpp

```diff
@@ -623,16 +623,15 @@ static void WindowScenarioselectScrollpaint(WindowBase* w, DrawPixelInfo* dpi, i
                         { window_scenarioselect_widgets[WIDX_SCENARIOLIST].width() - 45, y + 1 });
 
                     // Draw completion score
-                    const utf8* completedByName = "???";
-                    if (!String::IsNullOrEmpty(scenario->Highscore->name))
+                    u8string completedByName = "???";
+                    if (!scenario->Highscore->name.empty())
                     {
                         completedByName = scenario->Highscore->name;
                     }
-                    SafeStrCpy(buffer, completedByName, 64);
```

Comment:
Is `buffer` still used elsewhere after this?

Reply:
`buffer` is used earlier in the code (declared in line 603 and used in line 609). It doesn't serve a purpose after that anymore.

Reply:
I will scrape the other raw string uses eventually